lib/main_ltp.pm

```diff
@@ -93,12 +95,23 @@ sub loadtest_from_runtest_file {
         loadtest 'create_junkfile_ltp';
     }
 
+    if (!-d $unpack_path) {
+        mkdir($unpack_path, 0755);
+    }
```

Comment:
`mkdir($unpack_path, 0755) if (!-d $unpack_path);` would be better, but what about just `mkdir($unpack_path, 0755);` without condition? If directory doesn't exit, it will be created. If it exits, it will just continue. `mkdir` just returns different value (which is not needed to check).

Reply:
or just `mkdir -p` in `assert_script_run` could be used

Reply:
Reminder that `mkdir()` and `assert_script_run('mkdir ...')` create a directory on different machines.